app/models/form_element.rb

```diff
@@ -1,5 +1,6 @@
 class FormElement < ActiveRecord::Base
   belongs_to :form
+  has_paper_trail
```

Comment:
especially here - these come and go pretty fast in the form element editor. how would we even recover these?